.github/workflows/integration.yml

```diff
@@ -34,10 +36,26 @@ on:
   # manual trigger
   workflow_dispatch:
     inputs:
```

Comment:
The ordering of `dbt_core_branch`, `name`, `dbt_adapters_branch`, `adapter_branch` feels odd. I would group the branches either in descending or ascending order (e.g. `dbt_core_branch`, `dbt_snowflake_branch`, `dbt_adapters_branch`) and then put `name` either before or after that group. I would also make the arguments that go into the shell script are in the same order as the inputs to this workflow.

I think we might as well add `dbt-common` to this list as well. I know it's rare that we need to test against a feature branch of `dbt-common`, but that's what defaults are for.

Reply:
I was wondering about dbt_common... It is less common but you're right that we might as well handle it all.

Reply:
dbt-common added. Rearranged order of fields to "name", "adapter_branch", "dbt_adapters_branch", "dbt_core_branch", and "dbt_common_branch", because that feels like the natural order to me.